### PR TITLE
Material conversion

### PR DESCRIPTION
**Pull request by anandsinghparihar**

### CODE REVIEW

**Comment by rynomster** [comment]
I've created the develop branch and changed the base of this PR.
There seem to be quite allot of conflicts :(

**Comment by rynomster** [comment]
hey @anandsinghparihar, I'm sorry but this is the last time I'm going to ask you to resolve conflicts.. Could you please resolve for us, then I will merge into develop?

File: src/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
   <base href="/">
   <link rel="stylesheet" type="text/css" href="assets/index.css">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (rynomster) [inline src/index.html]: Please add this locally.. We can't make any remote calls due to privacy

File: karma.conf.js [hunk 2]
```diff
@@ -16,7 +16,8 @@ module.exports = function (config) {
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
     files: [
-      { pattern: './src/test.ts', watched: false }
+      { pattern: './src/test.ts', watched: false },
+      'node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css'
```
Comment (rynomster) [inline karma.conf.js]: what's this?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -55,6 +61,7 @@ export class AppComponent implements OnInit {
 
 
   createWallet() {
+    this.dialog.open(ModalsComponent, {disableClose: true, width: '100%', height: '100%'});
```
Comment (rynomster) [inline src/app/app.component.ts]: Please make sure we only make one service call, and not a dialogue and modal service call...

**Comment by rynomster** [comment]
Please fix the tests and lint and I will merge it, then we can fix it.
```
   src/app/wallet/send/send.component.html
      L133 |... mdTooltip="Copy" class="cursor-pointer" mdTooltip="Copy">content_copy</md-icon>
                                                       ^ Duplicate of attribute name [ mdTooltip ] was found. (attr-no-duplication)
```

File: src/app/app.component.ts (hunk 3 above)
Comment (rynomster) [reply src/app/app.component.ts]: We need to get rid of this

Comment (anandsinghparihar) [reply src/app/app.component.ts]: Yes I am going to open that material model inside  of Modal service

**Comment by rynomster** [comment]
If you do more commits, we can reopen this PR

File: src/index.html (hunk 1 above)
Comment (anandsinghparihar) [reply src/index.html]: Okay

File: karma.conf.js (hunk 2 above)
Comment (anandsinghparihar) [reply karma.conf.js]: I forgot to remove theme css

**Pull request merged**